_data/signed/pavel-rousar.yaml

```diff
@@ -0,0 +1,2 @@
+name: Pavel Roušar
+link: prousar@gmail.com
```

Comment:
Please prepend a mailto: for email links.

Reply:
Pushed an update.